src/components/TodoLoader/TodoLoader.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { TempTodoContext } from '../../TempTodoContext';
+
+export const TodoLoader: React.FC = () => {
```

Comment:
that's implemented fine, but note that in [the working example](https://mate-academy.github.io/react_todo-app-with-api/) we have `loader` on every action - deleting, adding (editing in future)

So in the next task try to do it in the same way